iot-hub/Samples/device/MessageReceiveSample/MessageReceiveSample.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Azure.Devices.Client.Samples
     public class MessageReceiveSample
     {
         private static readonly TimeSpan s_sleepDuration = TimeSpan.FromSeconds(5);
-        private static readonly TimeSpan s_receiveTimeout = TimeSpan.FromSeconds(10);
```

Comment:
This was too fast to let someone who isn't 100% ready in the portal or IoT Explorer.